src/lsst/cmservice/db/element.py

```diff
@@ -223,11 +228,11 @@ async def estimate_sleep_time(
             all_jobs = await self.get_jobs(session)
         for job_ in all_jobs:
             if job_.status == StatusEnum.running:
-                sleep_time = min(job_sleep, sleep_time)
+                sleep_time = max(job_sleep, sleep_time)
```

Comment:
Are you sure you want `max` here?  Meaning it will always sleep _at least_ `sleep_time`?

Reply:
Yes, sleep_time is less than job_sleep typically.